src/extension.ts

```diff
@@ -212,6 +238,19 @@ export function activate(context: vscode.ExtensionContext): void {
 
         node.treeItem.stopLogStream();
     });
+    initAsyncCommand(context, 'diagnostics.StartLogPointsSession', async (node: IAzureNode<SiteTreeItem>) => {
+        if (node) {
+            const client: WebSiteManagementClient = nodeUtils.getWebSiteClient(node);
+            const wizard = new LogPointsSessionAttach(outputChannel, client, node.treeItem.site, node.subscription);
+            await wizard.run();
+        }
+    });
+
+    initAsyncCommand(context, 'diagnostics.LogPoints.Toggle', async (uri: vscode.Uri) => {
+        logPointsManager.toggleLogpoint(uri);
+    });
+
+    context.subscriptions.push(vscode.commands.registerCommand('diagnostics.LogPoints.OpenScript', openScript));
```

Comment:
Use initCommand for this

(that way its tracked in our telemetry and errors are displayed properly)

Reply:
`initCommand()` only transfer 1 parameter to the cakkback. My `openScript` expects two parameters. Hence the trouble.

Filed https://github.com/Microsoft/vscode-azureappservice/issues/195. Will look into a fix when I got a chance.